docs/cookbook/creating_custom_dropdowns.md

```diff
@@ -0,0 +1,98 @@
+# Creating custom dropdowns
+
+In eZ Platform you are able to implement custom dropdowns anywhere in the system.
+Follow bellow steps to learn how to manipulate small component/library in order to do so. 
+
+## Prepare custom dropdown structure
+
+First prepare the HTML code structure in the following way:
+
+```html hl_lines="2"
+<div class="ez-custom-dropdown">
+    <select class="ez-custom-dropdown__select" hidden multiple>
+        <option value="1" selected>Option 1</option>
+        <option value="2">Option 2</option>
+        <option value="3">Option 3</option>
+        <option value="4">Option 4</option>
+    </select>
+    <div class="ez-custom-dropdown__wrapper">
+        <ul class="ez-custom-dropdown__selection-info"></ul>
+        <ul class="ez-custom-dropdown__items ez-custom-dropdown__items--hidden">
+            <li data-value="" class="ez-custom-dropdown__item" disabled>Select an option</li>
+            <li data-value="1" class="ez-custom-dropdown__item">Option 1</li>
+            <li data-value="2" class="ez-custom-dropdown__item">Option 2</li>
+            <li data-value="3" class="ez-custom-dropdown__item">Option 3</li>
+            <li data-value="4" class="ez-custom-dropdown__item">Option 4</li>
+        </ul>
+    </div>
+</div>
+```
+
+Highlighted line in the code above contains a hidden native `select` input. It stores values of selection.
+Input is hidden because custom dropdown replicates its functionality.
+Do not remove it. Removal would break the functionality of any submission form.
+
+![Dropdown expanded state](img/dropdown_expanded_state.jpg)
+
+## Generate <select> input
```

Comment:
I'm not sure this section is needed, becuase it brings some kind of mess to the documentation. The `<select>` input is already in the code sample above.

Reply:
Or maybe you wanted to show how to display a version with multiple selection.